app/assets/stylesheets/_addtocalendar.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ * AddToCalendar Base CSS
+ * http://addtocalendar.com
+ */
+
+/* Base */
+
+.addtocalendar var{
+    display: none;
+}
+
+.addtocalendar {
+    position: relative;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background, display, position